---
timestamp: '2025-01-16T13:46:29.696367'
level: INFO
issuer: instance_solver
event_type: metadata
data:
    log_file: /home/remote/u7899572/lifted-white-plan-domain-repair/exp_logs_anu/astar-unary
        relax-prec lp066/miconic__ps3-2-err-rate-0-5.yaml
    instance_id: miconic__ps3-2-err-rate-0-5
    search_algorithm: astar
    plan_length: 12
    domain_class: miconic
    instance_name: ps3-2-err-rate-0-5
    ground_plan:
    - (up f0 f1)
    - (board f1 p0)
    - (down f1 f0)
    - (depart f0 p0)
    - (up f0 f3)
    - (board f3 p2)
    - (down f3 f0)
    - (depart f0 p2)
    - (up f0 f4)
    - (board f4 p1)
    - (down f4 f0)
    - (depart f0 p1)
    lifted_plan:
    -   - up
        - f0
        - f1
    -   - board
        - ?f1
        - ?p0
    -   - down
        - ?f1
        - ?f0
    -   - depart
        - f0
        - ?p0
    -   - up
        - ?f0
        - ?f3
    -   - board
        - f3
        - ?p2
    -   - down
        - ?f3
        - f0
    -   - depart
        - ?f0
        - ?p2
    -   - up
        - f0
        - f4
    -   - board
        - f4
        - p1
    -   - down
        - ?f4
        - f0
    -   - depart
        - f0
        - p1


---
timestamp: '2025-01-16T13:46:30.212017'
level: INFO
issuer: instance_solver
event_type: ground_repair
data:
    repair_length: 0
    repair: ''


---
timestamp: '2025-01-16T13:47:13.910176'
level: ERROR
issuer: instance_solver
event_type: error
data: "An error occurred during A* search: 'Node' object has no attribute 'neighbours'.\
    \ Stack trace: Traceback (most recent call last):\n  File \"instance_solver.py\"\
    , line 104, in solve_instance\n    searcher.find_path(logger=logger, log_interval=log_interval)\n\
    \  File \"/home/remote/u7899572/lifted-white-plan-domain-repair/search_partial_grounding/search_algorithms.py\"\
    , line 75, in find_path\n    self.log_iteration_info(logger, iteration, open_list,\
    \ current_node, final=True, is_goal=True)\n  File \"/home/remote/u7899572/lifted-white-plan-domain-repair/search_partial_grounding/search_algorithms.py\"\
    , line 37, in log_iteration_info\n    \"current_node\": current_node.to_dict(include_state=False)\n\
    \  File \"/home/remote/u7899572/lifted-white-plan-domain-repair/search_partial_grounding/node.py\"\
    , line 234, in to_dict\n    \"num_neighbours\": len(self.neighbours),\nAttributeError:\
    \ 'Node' object has no attribute 'neighbours'\n"


---
timestamp: '2025-01-16T13:47:14.027339'
level: INFO
issuer: instance_solver
event_type: timer_seconds
data: '153.87'


---
timestamp: '2025-01-16T13:49:02.687536'
level: INFO
issuer: batch_solver
event_type: general
data: 'Subprocess finished successfully. Duration: 303.23 seconds'


